src/typescript.ts

```diff
@@ -21,20 +21,20 @@ export const genTypeObject = (obj: TypeObject, indent = '') => {
   }), indent, '{}', false)
 }
 
-export const genInterface = (name: string, contents?: TypeObject, options: GenInterfaceOptions = {}) => {
+export const genInterface = (name: string, contents?: TypeObject, options: GenInterfaceOptions = {}, indent = '') => {
```

Comment:
Shall we move `indent` to `GenInterfaceOptions` as a number? Similar to this: https://github.com/unjs/untyped/blob/main/src/generator/dts.ts#L80

Reply:
rebased to merge for 1.x let's reiterate api for v2.